Hymns/Browse/CategoryView.swift

```diff
@@ -9,7 +9,7 @@ struct CategoryView: View {
     var body: some View {
         VStack(alignment: .leading) {
             HStack {
-                Text(viewModel.category)
+                Text(viewModel.category).fixedSize(horizontal: false, vertical: true)
```

Comment:
Can you explain what the `.fixedSize(horizontal: false, vertical: true)` does? I'm looking at the browse page and none of them right now are cut off so far

Reply:
It keeps the text from truncating solving the issue that Emily posted. https://stackoverflow.com/questions/56593120/how-do-you-create-a-multi-line-text-inside-a-scrollview-in-swiftui/56604599#56604599